tests/cli/scanner/scanner_test.py

```diff
@@ -0,0 +1,20 @@
+"""Unit tests for the ostorlab scanner subcommand."""
+import sys
+
+import pytest
+from click import testing as click_testing
+
+from ostorlab.cli import rootcli
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="does not run on windows")
+def testRootCli_whenDaemonCommandIsProvided_runsBackground(mocker, requests_mock):
+    """Run CLI with daemon command."""
```

Comment:
Please update the docstring, add type hints and add the `\n` splitting the unit test into its 3 phases.